quantum/pointing_device_drivers.c

```diff
@@ -204,8 +202,8 @@ report_mouse_t pimorono_trackball_get_report(report_mouse_t mouse_report) {
 const pointing_device_driver_t pointing_device_driver = {
     .init       = pimironi_trackball_device_init,
     .get_report = pimorono_trackball_get_report,
-    .set_cpi    = NULL,
-    .get_cpi    = NULL
+    .set_cpi    = pimoroni_trackball_set_cpi,
+    .get_cpi    = pimoroni_trackball_get_cpi
```

Comment:
Should invocations of `driver.get_cpi()` and `driver.set_cpi(xxx)` need checks against NULL at their callsites?

Reply:
I asked about this on [discord]( https://discord.com/channels/440868230475677696/867530715192885269/923644316470091796) looks like it'll be dealt with in a future PR.